### PR TITLE
chore: support lib v0.36.2 binaries

### PR DESCRIPTION
**Pull request by eitsupi**

### CODE REVIEW

**Review by etiennebacher — APPROVED** [review]

**Comment by eitsupi** [comment]
Wow, what a quick review!
(This is the first binary library release support since switching to the arm64 macOS runner, so I'll wait until CI passes before merging just to be safe)

**Pull request merged**
